drivers/justswap.js

```diff
@@ -0,0 +1,39 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat, flatMap } = require('../lib/utils');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Justswap extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const { total_num: total } = await request('https://api.justswap.io/v2/allpairs?page_size=50&page_num=0');
+
+    const indexes = Math.floor(total / 50) + (total % 50 === 0 ? 0 : 1);
+    const all = Array.from(new Array(indexes), () => 1);
+
+    return flatMap(all, async (item, index) => {
+      const { data } = await request(`https://api.justswap.io/v2/allpairs?page_size=50&page_num=${index}`);
+
+      return Object.values(data).map((ticker) => new Ticker({
+        base: ticker.base_symbol,
+        baseName: ticker.base_name,
+        baseReference: ticker.base_id,
+        quote: ticker.quote_symbol,
+        quoteName: ticker.quote_name,
+        quoteReference: ticker.quote_id,
+        close: parseToFloat(ticker.price),
+        baseVolume: parseToFloat(ticker.base_volume) / 10 ** 6,
+        quoteVolume: parseToFloat(ticker.quote_volume) / 10 ** 6,
```

Comment:
I think it would be safe to use the `quote_decimal` they provide instead of the hardcoded six. I don't know if any quote currency in this API has more or less than 6, but just to be sure

Reply:
@nickpater I don't think that their base and quote decimal are correct. in some cases decimal is 13, 8 and so on.. Because of that, our test 'Quote volume and base volume are in the correct order' is failing.

Reply:
@nickpater Yes I did used base_decimal which caused 'Quote volume and base volume are in the correct order' error.

Reply:
Mh, can you check it again with both the `base_decimal` and `quote_decimal`, because the test passes for me when I use those!

Reply:
I just did and still crashes. I did remove old fixture and create new one before I started test. So old fixture is not causing this.